js/skin.js

```diff
@@ -287,8 +287,44 @@ var Skin = createReactClass({
     }
   },
 
+  /**
+   * This function changes a screen (pause/play) and
```

Comment:
Change the wording a little bit:
This function toggles the current screen (pause/play) and

Reply:
\+